advanced/dapps/react-dapp-v2/src/chains/bip122.ts

```diff
@@ -0,0 +1,28 @@
+import { NamespaceMetadata, ChainMetadata, ChainsMap } from "../helpers";
+
+export const BtcChainData: ChainsMap = {
+  "000000000933ea01ad0ee984209779baaec3ced90fa3f408719526f8d77f4943": {
+    id: "bip122:000000000933ea01ad0ee984209779baaec3ced90fa3f408719526f8d77f4943",
+    name: "BTC Signet",
+    rpc: ["https://api.devnet.solana.com"],
```

Comment:
Is this correct?

Reply:
nope. I will clean up the PR before its ready for review. Currently broadcasting transactions is manual as the wallet just signs them and returns them the signed hex the dapp